### ISSUE TITLE
Server update :-(

### ISSUE THREAD

**Issue by rupertleveneucd** [ttroy50/pyephember#3]
It seems this doesn't work since the app (and presumably the server) was updated a few days ago.

**Comment by ttroy50** [comment]
I haven't had a chance to look at it yet but will try to fix it next week.

**Comment by eiregrime** [comment]
I contacted EPH to ask what exactly they 'upgraded'. The response was: "The app is a complete overhaul" 
They made it even more difficult to use now as the 'fancy' app logs me out every time I try to use it. I'd love to be able to use it again via home assistant. Thanks in advance ttroy50... :)

**Comment by ttroy50** [comment]
I was able to capture the new API last night but still have to analyse it fully. It looks like most calls are changed but the core of the JSON structure is similar. 

I'll continue to look at ut tonight and hope to have a beta sometime over the weekend.

**Comment by ttroy50** [comment]
I've pushed changes to the `api-update` branch which work with the new API.

I've kept my basic python interface the same but most of the returned dictionaries are different. The API.md docs are updated with that info.

To make it easier to access some info I've added helper functions to get some of the common variables from a returned zone structure.

It would be nice to have others give this a quick test to make sure things are working for your use cases. Some basic tests can be done by using / modifying the `example.py` script in the root folder to run some calls against your account.

If you use this with home-assistant, I'll be updating that later today but won't be able to push those changes until I've fully tested and released this library to pypi.
